### PR TITLE
Modified the KMEA keyboard.html to use the new `keyman` JS namespace

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses the KMEA portion of #349

Modify keyboard.html to use the new `keyman` JS namespace instead of `tavultesoft.keymanweb`

### CODE REVIEW

File: android/KMEA/app/src/main/assets/keyboard.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
     // Query KMW if a given keyboard uses chiral modifiers.
     function setIsChiral(keyboardProperties) {
       var name = typeof(keyboardProperties.internalName) == "undefined" ? keyboardProperties.keyboardName : keyboardProperties.internalName;
-      var isChiral = tavultesoft.keymanweb.isChiral(name);
+      var isChiral = keymanweb.isChiral(name);
```
Comment (jahorton) [inline android/KMEA/app/src/main/assets/keyboard.html]: I assume `KeymanWeb` is a global here?

Comment (jahorton) [reply android/KMEA/app/src/main/assets/keyboard.html]: `keymanweb`- autocorrect messed it up.

File: android/KMEA/app/src/main/assets/keyboard.html [hunk 2]
```diff
@@ -106,22 +108,22 @@
     }
 
     function resetContext() {
-      tavultesoft.keymanweb.resetContext();
+      keymanweb.resetContext();
```
Comment (jahorton) [inline android/KMEA/app/src/main/assets/keyboard.html]: Likewise here?

File: android/KMEA/app/src/main/assets/keyboard.html (hunk 1 above)
Comment (darcywong00) [reply android/KMEA/app/src/main/assets/keyboard.html]: good catch. I need to change those to `kmw`.

**Pull request merged**